File: dotnet/cafdemalihapa/Directories/Repository/Dotnet/Cafdemalihapa/Scripts/EnvironmentVariablesSource/EnvironmentVariablesSourceDirectory.cs
namespace cafdemalihapa.Directories.Repository.Dotnet.Cafdemalihapa.Scripts.EnvironmentVariablesSource
{
    public class EnvironmentVariablesSourceDirectory(
        ScriptsDirectory scriptsDirectory
        )
    {
        public string GetPath()
        {
            var environmentVariablesSourceDirectory =
                Path.Combine(scriptsDirectory.GetPath(), "environment-variables-source");

            return environmentVariablesSourceDirectory;
        }
    }
}

File: dotnet/cafdemvers/zzq-cafdemwimu/cafdemwimu.console/Directories/Repository/Dotnet/Cafdemalihapa/Scripts/EnvironmentVariablesSource/EnvironmentVariablesSourceDirectory.fs
namespace cafdemwimu.console.Directories.Repository.Dotnet.Cafdemalihapa.Scripts.EnvironmentVariablesSource

open System.IO
open cafdemwimu.console.Directories.Repository.Dotnet.Cafdemalihapa.Scripts

type EnvironmentVariablesSourceDirectory(scriptsDirectory: ScriptsDirectory) =
    member _.GetPath() =
        let environmentVariablesSourceDirectory =
            Path.Combine(scriptsDirectory.GetPath(), "environment-variables-source")
        environmentVariablesSourceDirectory
